docs/reference/cli/search.md

```diff
@@ -17,38 +17,39 @@ query [--include-edges] [--explain] <query>
 
 ### Parameters
 
-| Parameter | Description      | Required? | Default Value |
-| --------- | ---------------- | --------- | ------------- |
-| `query`   | Query to execute | ✔️        | N/A           |
+| Parameter | Description             | Required? | Default Value |
+| --------- | ----------------------- | --------- | ------------- |
+| `query`   | Search query to execute | ✔️        | N/A           |
```

Comment:
"Search to execute"? On Google you're running a search, not a search query.

Reply:
@lloesche Hm, any suggestions for renaming the parameter as well?  `search` isn't a super descriptive parameter name for the `search` command...

Reply:
I'm going to merge this PR first since there are some merge conflicts with what I'm working on, but we can come back and fix the language here after we come to a decision on what is best :)